scripts/create_report.R

```diff
@@ -71,6 +71,11 @@ if(!require(RColorBrewer)) {
 	library(RColorBrewer, lib.loc=local.lib)
 }
 
+if(!require(yaml)) {
+        install.packages("yaml", repos = c('http://rforge.net', 'http://cran.rstudio.org'), type = 'source', lib=local.lib)
+        library(RColorBrewer, lib.loc=local.lib)
```

Comment:
should say `library(yaml, lib.loc=local.lib)` ?

Reply:
good catch thanks.